CONTRIBUTING.md

```diff
@@ -1,7 +1,7 @@
 # Contributing
 
 Thanks for the interest! See the [contributor documentation][contribute]
-to get started, and reach out on [Discord][discord] if you get stuck.
+to get started, and reach out on [Discord] if you get stuck.
```

Comment:
I think we still need this. It's linking to the discord link below. Alternatively, we could inline the links.

Reply:
ooo, fancy!